Core/Services/CarService.cs

```diff
@@ -67,12 +67,11 @@ public IQueryable<CarDTO> GetAllUserCars(string userId)
 
         private bool IsCarExist(Car newCar)
         {
-            return _carRepository
-                .Query()
-                .Any(c => 
-                    c.RegistrationNumber == newCar.RegistrationNumber ||
-                    c.Vin == newCar.Vin ||
-                    c.TechnicalPassport == newCar.TechnicalPassport);
+            return await _carRepository
+                .AnyAsync(new GetCarWithMainCredentials(
+                    newCar.RegistrationNumber,
+                    newCar.Vin,
+                    newCar.TechnicalPassport));;
```

Comment:
Incorrect. Please make or PK(RegistrationNumber, Vin, TechnicalPassport) if all these 3 fields make row record unique

Reply:
We will made this in further pr`s